fipy/variables/cellVariable.py

```diff
@@ -211,8 +211,8 @@ def cellVolumeAverage(self):
         """
 
         if not hasattr(self, 'volumeAverage'):
-            from fipy.variables.cellVolumeAverageVariable import _CellVolumeAverageVariable
-            self.volumeAverage = _CellVolumeAverageVariable(self)
+            volumes = CellVariable(mesh=self.mesh, value=self.mesh.cellVolumes)
```

Comment:
Why are you creating another CellVariable here?

Reply:
It's been like that in `_CellVolumeAverageVariable` for about 11 years, but you're right